test/IDE/merge_local_types.swift

```diff
@@ -29,18 +29,18 @@ public func toMerge() {
 
 //CHECK-DAG: 16LocalTypesMerged7toMergeyyF14SingleFuncEnumL_O
 //CHECK-DAG: 16LocalTypesMerged10singleFuncyyF06SingleE4EnumL_O
-//CHECK-DAG: 16LocalTypesMerged21singleDefaultArgument1iySi_tFfA_SiycfU_06SingleeF5ClassL_C
```

Comment:
Same for this file (`iy` is correct)

Reply:
Oh those were not intentional by me, sorry for missing those... thanks!